release.nix

```diff
@@ -4,7 +4,7 @@
 let
   native-reflex-platform = reflex-platform-fun { __useNewerCompiler = true; };
   inherit (native-reflex-platform.nixpkgs) lib;
-  systems = ["x86_64-linux" "x86_64-darwin"];
+  systems = ["x86_64-linux"];
```

Comment:
This is a bummer, but I think it is fine for now because https://ci.obsidian.systems/ is in maintenance mode, and other things (like reflex itself) will get this.